test/Psy/Test/CodeCleaner/ImplicitReturnPassTest.php

```diff
@@ -33,6 +33,7 @@ public function implicitReturns()
         return array(
             array('4',     'return 4;'),
             array('foo()', 'return foo();'),
+            array('exit()', 'die;'),
```

Comment:
Thought it makes sense to also add a new item to the data provider to check that no implicit return is added now.